otx/algorithms/classification/adapters/deep_object_reid/utils/monitors.py

```diff
@@ -0,0 +1,100 @@
+"""Monitors for deep_object_reid tasks."""
+
+# Copyright (C) 2022 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+import abc
+
+from otx.api.utils.argument_checks import check_input_parameters_type
+
+
+class IMetricsMonitor(metaclass=abc.ABCMeta):
```

Comment:
I can see that these interfaces (IStopCallback as well) are not being exported. Do you think we can just keep the concrete implementations if there are not other derived classes for it and if it is not being used for type checking. Can be part of a different PR though.

Reply:
@supersoob Is this the code used? If used code, when will it be removed?

Reply:
Thanks for pointing it out. This is abstractclasses for other classes inside monitors, not be imported by other files. I think it can be removed.